openedx/adg/lms/utils/decorators.py

```diff
@@ -0,0 +1,26 @@
+"""
+A utility for custom decorators
+"""
+import functools
+
+from django.conf import settings
+from django.dispatch import receiver
+
+
+def suspendingreceiver(signal, **decorator_kwargs):
```

Comment:
Didn't we do this in tests config file?

Reply:
No, this is different. Using `suspendingreceiver ` instead of `receiver` will trigger fake/empty `receiver` on signal call.